src/main.rs

```diff
@@ -1,15 +1,12 @@
 #![recursion_limit = "128"]
-extern crate fern;
-#[macro_use]
-extern crate log;
-extern crate screeps;
 #[macro_use]
 extern crate stdweb;
```

Comment:
Following https://doc.rust-lang.org/stable/edition-guide/rust-2018/macros/macro-changes.html

This should be switched to `use stdweb::js` .

Reply:
Right, guess this should probably change too!

It currently requires importing a helper macro too, but I've filed an issue for that: https://github.com/koute/stdweb/issues/347